apps/export/entries/excel_exporter.py

```diff
@@ -36,7 +37,8 @@ class ColumnsData:
             ] if self.modified_excerpt_exists else ['Excerpt'],
         }
 
-    def __init__(self, export_object, entries, columns=None, decoupled=True, project_id=None, is_preview=False):
+    def __init__(self, export_object, entries, project, columns=None, decoupled=True, is_preview=False):
```

Comment:
Since project object is not used inside, why was project object passed instead of project_id? Just out of curiosity.